chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+      if (popup) {
+        popup.init_popup();
+        self.updateData();
+      }
+    });
+  }
+
+  /**
+   * Save settings
+   */
+  saveSettings() {
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)